src/main/java/org/broadinstitute/hellbender/utils/plotter/SegmentedCopyRatioPlotter.java

```diff
@@ -21,16 +24,18 @@ private CopyRatioSegmentedPlotter() {
      * @param log Input tangent file has had a log2 transform applied
      * @param useSexChromosomes plot results for X and Y chromosomes
      */
-    public static void writeSegmentedCopyRatioPlot(final String sampleName, final String tnFile, final String preTnFile,
-            final String segmentsFile, final String outputDir, final Boolean log, final Boolean useSexChromosomes) {
-        String logArg = log ? "TRUE" : "FALSE";
-        String schr = useSexChromosomes ? "TRUE" : "FALSE";
+    public static void writeSegmentedCopyRatioPlot(final String sampleName, final File tnFile, final File preTnFile,
+                                                   final File segmentsFile, final String outputDir,
+                                                   final Boolean log, final Boolean useSexChromosomes) {
+        final String logArg = log ? "TRUE" : "FALSE";
+        final String schr = useSexChromosomes ? "TRUE" : "FALSE";
         final RScriptExecutor executor = new RScriptExecutor();
         //This leads to the R statement source("CNV_plotting_library.R") before the main script runs
-        executor.addScript(new Resource(CNV_PLOTTING_R_LIBRARY, CopyRatioSegmentedPlotter.class));
-        executor.addScript(new Resource(COPY_RATIO_PLOTTING_R_SCRIPT, CopyRatioSegmentedPlotter.class));
+        executor.addScript(new Resource(CNV_PLOTTING_R_LIBRARY, SegmentedCopyRatioPlotter.class));
+        executor.addScript(new Resource(COPY_RATIO_PLOTTING_R_SCRIPT, SegmentedCopyRatioPlotter.class));
         /*--args is needed for Rscript to recognize other arguments properly*/
-        executor.addArgs("--args", "--sample_name="+sampleName, "--targets_file="+tnFile, "--pre_tn_file="+preTnFile, "--segments_file="+segmentsFile,
+        executor.addArgs("--args", "--sample_name="+sampleName, "--targets_file="+tnFile.getAbsolutePath(),
+                "--pre_tn_file="+preTnFile.getAbsolutePath(), "--segments_file="+segmentsFile.getAbsolutePath(),
```

Comment:
do you need the `getAbsolutePath()` here? it seems to work just by passing the file as a parameter(as in `ACNVPlotter`), since `addArgs()` converts all arguments into strings anyway